class.tx_mksanitizedparameters.php

```diff
@@ -354,8 +354,9 @@ private function injectCommonRulesFromCurrentIntoNextLevelIfNotSet(
     }
 
     /**
-     * @param array $rules
-     * @param mixed $defaultRules
+     * @param array $rulesForValue
+     * @param mixed $allRules
```

Comment:
You could force $allRules to be an array by adding the appropriate type hint in the parameter signature of the method. It can never be something else if the code of the method should work as the variable is accessed as an array.

Reply:
You're right. That i will do!